### PR TITLE
[Hint Refactor] Changed hint messages to use a temp new style

### PR DESCRIPTION
**Pull request by MeltedPixel**
## What type of PR is this.
1. [ ] Bug
2. [X] Enhancement

### What have you changed and why?
Information:
    
The current hint messages were bland and sometimes hard to read. With the changes made, the hint messages are now more clear and branded.

Additionally, this leaves us in a good spot to make any modifications we need in a single file rather then each hint message itself.

This means when Doom is finally ready, we can easily move to his idea of hint messages as long as the call format stays the same.

### Please specify which Issue this PR Resolves.
closes N/A

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes:

Please check for anything missed or wrong. There's a TON of hint messages changed.

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
@Spoffy 
This is cleared for merging.

**Comment by MeltedPixel** [comment]
Would you guys like me to go through and fix the conflicting files? One that we should take notice about is saveLoop.sqf. It's no longer a file. The save loop is now a function within the Save Functions folder.

**Comment by Bob-Murphy** [comment]
Normally this is done by Spoffy when he merges it.
Nevertheless, I wouldn't have a problem with you fixing it so he doesn't have to do it but better clear that up with him.

**Comment by Spoffy** [comment]
If you don't mind @MeltedPixel, it'd be appreciated! Save me some more time to work on the Garrison review :)

File: A3-Antistasi/functions/Save/fn_saveLoop.sqf [hunk 1]
```diff
@@ -1,12 +1,12 @@
 if (hasInterface) then {
-	if (!isNil "savingClient" && {savingClient}) exitWith {["Save Game", "Your personal stats are being saved"] call A3A_fnc_customHint;};
+	if (!isNil "savingClient" && {savingClient}) exitWith {hint "Your personal stats are being saved"};
```
Comment (Spoffy) [inline A3-Antistasi/functions/Save/fn_saveLoop.sqf]: Was resetting this intentional?

File: A3-Antistasi/functions/Save/fn_saveLoop.sqf [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 	[] call A3A_fnc_savePlayer;
 };
 
 //Server only from here on out.
 if (!isServer) exitWith {};
 
-if (savingServer) exitWith {["Save Game", "Server data save is still in progress"] remoteExecCall ["A3A_fnc_customHint",theBoss]};
+if (savingServer) exitWith {"Server data save is still in progress" remoteExecCall ["hint",theBoss]};
```
Comment (Spoffy) [inline A3-Antistasi/functions/Save/fn_saveLoop.sqf]: Was resetting this intentional?

File: A3-Antistasi/functions/Save/fn_saveLoop.sqf [hunk 3]
```diff
@@ -206,6 +206,6 @@ _controlsX = controlsX select {(sidesX getVariable [_x,sideUnknown] == teamPlaye
...
-[petros,"hint",_saveHintText, "Save Game"] remoteExec ["A3A_fnc_commsMP", 0];
+_saveHintText = format ["Savegame Done.\n\nYou won't lose your stats in the event of a game update.\n\nRemember: if you want to preserve any vehicle, it must be near the HQ Flag with no AI inside.\nIf AI are inside, you will save the funds you spent on it.\n\nAI will be refunded\n\nStolen and purchased Static Weapons need to be ASSEMBLED in order to be saved. You can save disassembled Static Weapons in the ammo box.\n\nMounted Statics (Mortar/AA/AT squads) won't get saved, but you will be able to recover the cost.\n\nSame for assigned vehicles more than 50m away from HQ.\n\n%1 fund count:\nHR: %2\nMoney: %3 €",nameTeamPlayer,_hrBackground,_resourcesBackground];
+[petros,"hint",_saveHintText] remoteExec ["A3A_fnc_commsMP", 0];
```
Comment (Spoffy) [inline A3-Antistasi/functions/Save/fn_saveLoop.sqf]: Was resetting this intentional?

**Pull request merged**